eo-runtime/src/test/eo/org/eolang/rust-tests.eo

```diff
@@ -256,10 +257,10 @@
     use eo_env::EOEnv;
     use eo_env::eo_enum::EO;
     use eo_env::eo_enum::EO::{EOInt};
-    pub fn foo(env: &mut EOEnv) -> EO {
-      let v = env.find("$.^.a") as u32;
-      let _bytes = env.dataize(v).unwrap();
-      EOInt(v as i64)
+    pub fn foo(env: &mut EOEnv) -> Option<EO> {
+      let v = env.find("$.^.a")?;
+      let _bytes = env.dataize(v)?;
```

Comment:
@levBagryansky Should there also be 4 spaces missing at the beginning of the line with the code?